mbus/nats_handler.go

```diff
@@ -28,12 +29,10 @@ import (
 )
 
 const (
-	responseMaxLength           = 1024 * 1024
-	natsHandlerLogTag           = "NATS Handler"
-	natsConnectionTimeout       = 5 * time.Minute
```

Comment:
This 5 minute timeout was introduced to avoid agents crashing when switching from bosh retry to nats retry. 

https://github.com/cloudfoundry/bosh-agent/commit/9b88d9fffeab52221aa0bbb5c633c9a9876cbf91

The Issues with the `agent timed out` on deployments after a bosh create-env appeared right after.